mesa/experimental/jupyter_viz.py

```diff
@@ -281,7 +221,17 @@ def portray(g):
             out["c"] = c
         return out
 
-    space_ax.scatter(**portray(space))
+
+    # figure(figsize=(50, 50), dpi=80)
```

Comment:
Needs to be removed.

Reply:
introduced by automated merge, should be resolved